lib/integrations/awsoidc/listdatabases.go

```diff
@@ -233,7 +234,8 @@ func listDBClusters(ctx context.Context, clt ListDatabasesClient, req ListDataba
 		// All the members have the same network information, so picking one at random should not matter.
 		clusterInstance, err := fetchSingleRDSDBInstance(ctx, clt, req, aws.ToString(db.DBClusterIdentifier))
 		if err != nil {
-			return nil, trace.Wrap(err)
+			log.DebugContext(ctx, "Failed to fetch RDS instance", "error", err)
```

Comment:
Should we only log/continue when we know the error was `instance not found`? 
Can we also include the RDS identifier here?
```suggestion
			log.InfoContext(ctx, "Skipping RDS Cluster because it has no instances.", "rds_dbclusterid", db.DBClusterIdentifier, "error", err)
```

Reply:
> Should we only log/continue when we know the error was instance not found?
> Can we also include the RDS identifier here?

yeah I think these are both good points.

Updated to only skip when it's not found, and changed to info level log: [diff](https://github.com/gravitational/teleport/compare/64e12dfd43c04846ca43b6c5786299d3f68b3790..22b232f3b1626217844e070cb22f7f4fc927c2bc)